orchestrators/FinanceOperations/src/Coordinators/BudgetTrackingCoordinator.php

```
@@ -93,12 +94,14 @@ public function checkBudgetAvailable(BudgetCheckRequest $request): BudgetCheckRe
-                'tenantId' => $request->tenantId,
-                'budgetId' => $request->budgetId,
-                'amount' => (string)$request->amount,
-                'costCenterId' => $request->options['cost_center_id'] ?? null,
-            ]);
+            $ruleResult = $this->budgetAvailableRule->check(
+                RuleContext::forBudgetAvailability(
+                    tenantId: $request->tenantId,
+                    budgetId: $request->budgetId,
+                    amount: (string) $request->amount,
+                    costCenterId: $request->options['cost_center_id'] ?? null,
+                )
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Nexus\FinanceOperations\DTOs\RuleContext' in method 'checkBudgetAvailable'. (undefined)

(StaticAccess)